RadixWallet/Features/HomeFeature/Coordinator/Home.swift

```diff
@@ -7,21 +7,25 @@ public struct Home: Sendable, FeatureReducer {
 		// MARK: - Components
 		public var babylonAccountRecoveryIsNeeded: Bool
```

Comment:
Is this flag still needed with this PR?

Reply:
That was what I was gonna ask you! What so you think? Do we wanna keep track of accounts controlled by BDFS not passing FULL check (public keys validation)? Then we need to retain that state

Reply:
`Do we wanna keep track of accounts controlled by BDFS not passing FULL check (public keys validation)` That is the reason it was kept around. I recall we had this discussion at the time when I have found this flag not being used.

Reply:
resolved in Huddle, we will remove this property, not needed at this point, we will treat all mnemonics the same.

Reply:
resolved over huddle